recipes/debreak/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.2" %}
+
+package:
+  name: debreak
+  version: {{ version }}
+
+source:
+    url: https://github.com/ChongLab/DeBreak/archive/refs/tags/debreak-v{{ version }}.tar.gz
+    sha256: 6162786bc60dc995d25790da6f806ed25bcfa50a7b10dc6e8404cacba7ce36df
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
+  run:
+    - python =2.7
+    - minimap2 =2.15
+    - pysam =0.16.0.1
+    - samtools =1.9
+    - wtdbg =2.5
```

Comment:
Do we really need these exact versions of these dependencies and now say "minimap2 >=2.15"?

Reply:
We have not tested the newer versions. Should be Okay. But it's not guaranteed.